src/screens/CreatePost/elements/useHastagMention.js

```diff
@@ -6,14 +6,14 @@ const useHastagMention = (initial = '') => {
   const [formattedText, setFormattedText] = React.useState(initial);
   const [hashtags, setHashtags] = React.useState([]);
 
-  const handleStateHashtag = (text, setHashtagState = null) => {
-    const hashtagMention = handleHastagMention(text, hashtags, setHashtagState);
+  const handleStateHashtag = (text) => {
+    const hashtagMention = handleHastagMention(text, hashtags);
     setFormattedText(hashtagMention);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `setHashtagState` parameter from the `handleStateHashtag` function seems to be a good change, as it was not being used in the previous implementation. However, please ensure that this function is not called elsewhere with this additional argument.



<!-- This is an auto-generated comment by OSS CodeRabbit -->